dev/com.ibm.ws.anno/src/com/ibm/wsspi/anno/info/MethodInfo.java

```diff
@@ -133,12 +133,12 @@ public interface MethodInfo extends Info {
     /**
      * <p>Answer the in-order parameter annotation collections of
      * parameters of this method.</p>
-     * 
+     *
      * @return The in-order parameter annotation collections of parameters
      *         of this method.
      */
-    public List<List<? extends AnnotationInfo>> getParameterAnnotations();
```

Comment:
This build failed because this is an SPI so you can't just delete a method and add a new one without updating version.  I think you can leave it the way it was in order to not require a SPI version update.

Reply:
I see you added it back and also added all the other `@Deprecated` methods in the latest commits.  Only this one method needed to be added back since it is the only one that is on a SPI interface.

Reply:
Fixing MethodInfo (in anno) to be as it was before.  The update is not needed here, just in annocache.
(I missed that the prior comment was re: the copy under "anno".)